Pear/Controller/HomeViewController.swift

```diff
@@ -15,6 +15,7 @@ import AppDevAnnouncements
 class HomeViewController: UIViewController {
 
     // MARK: - Private View Vars
+    private var feedbackMenuView: FeedbackView!
```

Comment:
If this is not always shown, then it should be optional `?`

In truth, this is probably not the cleanest way to go about this. I'm not sure exactly what this looks like, but judging from the implementation, it looks like a popup. If so, it might be cleaner to extract the popup into another view controller that manages its own presentation and dismissal in the same way as an action sheet on iPad.